Lib/defconQt/objects/glyphDialogs.py

```diff
@@ -12,20 +12,23 @@ class GotoDialog(QDialog):
     def __init__(self, currentGlyph, parent=None):
         super(GotoDialog, self).__init__(parent)
         self.setWindowModality(Qt.WindowModal)
-        self.setWindowTitle("Go to…")
+        self.setWindowTitle(self.tr("Go to…"))
         self.font = currentGlyph.getParent()
         self._sortedGlyphs = self.font.unicodeData.sortGlyphNames(
             self.font.keys(), self.alphabetical)
 
         layout = QGridLayout(self)
-        self.glyphLabel = QLabel("Glyph:", self)
+        self.glyphLabel = QLabel(
+            QCoreApplication.translate("GotoDialog", "Glyph:"), self)
```

Comment:
The goto dialog gets repurposed by that other dialog, whatsitcalled, which mucks with self.tr. Translations would appear in the goto dialog but not in the other.